pylint_odoo/checkers/no_modules.py

```diff
@@ -324,6 +369,15 @@ def visit_call(self, node):
                                 '_' + argument.arg + '_'):
                         self.add_message('method-' + argument.arg,
                                          node=argument_aux)
+                    elif (argument.arg in
+                            self.config.deprecated_field_parameters):
```

Comment:
Non blocking, but you could remove a few lines by using an intermediary variable:

```
deprecated = self.config.deprecated_field_parameters
```